imap_processing/idex/idex_l1a.py

```diff
@@ -515,55 +467,54 @@ def process(self) -> xr.Dataset:
         dataset : xarray.Dataset
             A Dataset object containing the data from a single impact.
         """
-        # Create object for CDF attrs
+        # Create an object for CDF attrs
         idex_attrs = self.cdf_attrs
 
-        # Gather the huge number of trigger info metadata
+        # Gather the huge amount of metadata info
         trigger_vars = {}
-        for var, value in self.trigger_values.items():
-            trigger_desc = TRIGGER_DESCRIPTION_DICT[var]
+        for var, value in self.telemetry_items.items():
             trigger_vars[var] = xr.DataArray(
                 name=var,
                 data=[value],
                 dims=("epoch"),
-                attrs=idex_attrs.get_variable_attributes(trigger_desc.name),
+                attrs=idex_attrs.get_variable_attributes(var),
             )
 
         # Process the 6 primary data variables
         tof_high_xr = xr.DataArray(
             name="TOF_High",
             data=[self._parse_high_sample_waveform(self.TOF_High_bits)],
-            dims=("epoch", "time_high_ssr_dim"),
+            dims=("epoch", "time_high_sr_dim"),
             attrs=idex_attrs.get_variable_attributes("tof_high_attrs"),
         )
         tof_low_xr = xr.DataArray(
             name="TOF_Low",
             data=[self._parse_high_sample_waveform(self.TOF_Low_bits)],
-            dims=("epoch", "time_high_sr"),
+            dims=("epoch", "time_high_sr_dim"),
```

Comment:
Good question. I am not sure, and I think we should go with what makes sense to you, but if it remains confusing we could ask Bryan about it since he wrote it.

Reply:
Good call - Ill ask Bryan

Reply:
Bryan is on vacation but I changed all of the waveform arrays to use coordinates instead. After learning more about this and looking at other instruments l1a datasets I feel confident in this update.